standard-app/example.values.yaml

```diff
@@ -1,6 +1,10 @@
 image: us-central1-docker.pkg.dev/cloudkite-infra-ops/cloudkite-docker-images/example-image
 tag: example_tag
-
+pvcs:
```

Comment:
Please put a comment here about what `pvcs` are.

Reply:
This has been added, please review.